pkg/chains/formats/sbom/sbom.go

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
Wondering if this should go into `slsa/internal/sbom`?

Reply:
SBOM is not related to the SLSA Provenance format.